ShareContacts/explainer.md

```diff
@@ -0,0 +1,91 @@
+* This document status: **Active**
+* Expected venue: [W3C Web Incubator Community Group](https://wicg.io/)
+* **Current version: this document**
+
+## Introduction
+
+For modern web applications like social networks and instant messaging, contacts are a fundamental part of the app itself. These contacts are specfic to the application, but the platform/OS can benefit from exposing them in different people related surfaces. Contact Sharing for installed web applications allows web apps to surface contacts in relevant UX on the underlying OS. One example can be sharing. Different platforms have different share UX, and some of these UXs include a list of frequent contacts that make sharing content faster with people the user interacts with frequently.
+
+## Goals
+
+* Allow installed web applications to participate in contacts/people UX across different platforms.
+
+## Non-goals
+
+*
+
+## Use Cases
+
+### Integrating with platform's Share UX
+
+A user decides to share content and opens the Share UX on the platform. They are presented with their frequent contacts on several apps. They can click on a contact to share the content with that contact. Overall, this represents a faster an easier way to share content. This is also behaviour that is present in several desktop and mobile platforms already. It saves the steps of having to open the app and look for the contact you will be sharing with. The following image shows how the share UX shows contacts on Windows and Android. Notice that each contact is associated to a specific application, as represented by a small app-icon on the bottom-right corner of the displayed contact.
+
+![Share UX on multiple platforms](contactShare.png)
+
+## Proposed solution
+
+The proposed solution is a promise-based method that works in scope of an installed web application to share a list of contacts with the underlying platform. This gets the contacts-per-app into the OS. The other part of the feature is akin to Web Share Target, except it is targeted directly at a specific contact.
+
+### Providing Contacts to the platform
+
+`navigator.shareContacts(<contacts>);`
+
+The method receives contacts as a list of JSON objects. Each object must have at least an `id`, `firstName` and `displayPicture`. Optionally, the contact can have a `lastName`, `rank` and a list of `phones` and `emails`.
+
+#### Contact detailed fields
+
+* `id`: unique id for the contact.
+* `firstName`: Contact's first name(s).
+* `displayPicture`: url to the display picture of the contact.
+* `lastName`: Contact's last name(s).
+* `rank`: integer that defines the rank or precedence that a contact has in the list. This could be used by the platform to display top contacts first.
+* `phones`: list of strings that represent phone numbers.
```

Comment:
For consistency with the ContactInfo object, what about 'tels'. Or maybe naming these singularly, and simply accepting a list?
'tel'
'email'